### PR TITLE
Project Complete

### PR DESCRIPTION
**Pull request by s0kil**

### CODE REVIEW

**Review by Forison — COMMENTED** [review]
#### Pull request under review by Addo

File: index.html [hunk 1]
```diff
@@ -1,75 +1,101 @@
...
+      <div class="sign-in-action">
+        <p>Already have an Intuit account?</p>
+        <a href="#">I forgot my user ID or password.</a>
+      </div>
+      <button type="button">
+          <span class='lock-icon'></span>
+          <p>Sign In</p>
+      </button>
+    </div>
+    <div class="main">
```
Comment (Forison) [inline index.html]: #### Use the```< main>``` Html tag instead of having it on as a class to enhance semantics🙂

File: index.html [hunk 2]
```diff
@@ -1,75 +1,101 @@
...
+        <!--Container for the form in the middle of the page------------>
+        <div class="forms-container">
+          <section id="form-body">
+            <h2>Create an Intuit Account</h2>
+            <p class="subtitle">
+              One account for everything Intuit, including Mint.
+            </p>
+            <a href="#">Learn More</a>
+            <form class="sign-up-form" action="index.html" method="post">
+              <label for="e-mail">
```
Comment (Forison) [inline index.html]: Let us use ```<label>``` tag as their name suggests. ie `<label for="email">email</label> <input type="email>` You may the wrap them inside one ```div``` or whichever means you need to make them align the way you want. I believe that would be more semantic than it is now.🙂
> use this approach in the other related instances

**Review by Forison — CHANGES_REQUESTED** [review]
# Hi Guys🙂,
You have done a good job so far.
Let us make these changes to have the project approved💪.
* [x] Let us fix Html validator error returned by the[ validator ](https://validator.w3.org) after uploading the index file.🙂
* [x] Let us set up and configure linters as required.
Read this [article](https://github.com/microverseinc/linters-config#linters-config).
* [x] Let us add a descriptive readme to the repo.
Your readme file is supposed to be descriptive enough(readme has to describe the project you have worked on in detail so that anyone who takes your project will have a fair view of what the project is about when they view the readme)
* [x] Let us make use of semantic Html in the project.
Please read this [article](https://www.w3schools.com/html/html5_semantic_elements.asp) for clarification on the Topic.
* [x] Let us remove the unused selection form our CSS file. (obsolete selections must not be committed) check CSS line ```207 - 209```
>I see just two commits, one each from each of you🤔. (always remember to commits every major addition)🙂

## That is it, guys make these changes and submit the project for review.🙂
Feel free to contact me on [LinkedIn](https://www.linkedin.com/in/forison-addo-558201159/), [Twitter](https://twitter.com/ADDOBOAKYE1) or slack for further clarification if necessary. I would be happy to assist you.
##### Happy coding🙂

**Review by suretrust — APPROVED** [review]
Hi @s0kil and @Kasuhira,

### Amazing job guys! :raised_hands: Let's get this merged. :+1: 

_Good luck with your future projects and happy coding!_ :tada:

**Pull request merged**